auth-service/src/main/java/com/artsem/api/authservice/util/KeycloakEvent.java

```diff
@@ -0,0 +1,11 @@
+package com.artsem.api.authservice.util;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum KeycloakEvent {
+    UPDATE_PASSWORD("UPDATE_PASSWORD");
+    private final String event;
```

Comment:
Обычно между самими перечислениями енам и полями оставляют пустую строку
```suggestion
    UPDATE_PASSWORD("UPDATE_PASSWORD");
    
    private final String event;
```